v3/src/pages/docs/installation/aspnet-apps-workflow-server-and-designer.md

```diff
@@ -22,24 +22,24 @@ dotnet new web -n "WorkflowsApp" -f net7.0
 CD into the project's root directory and add the following packages:
 
 ```shell
-cd WorkflowServer.Api
-dotnet add package Elsa
-dotnet add package Elsa.Http
-dotnet add package Elsa.Workflows.Api
-dotnet add package Elsa.Workflows.Designer
+cd WorkflowsApp
```

Comment:
or keep `WorkflowServer.Api` but the folder name has to be the same than the app created previously.